src/swap/components/SwapProvider.tsx

```diff
@@ -46,7 +46,7 @@ export function SwapProvider({
   const { address } = useAccount();
   // Feature flags
   const { useAggregator } = experimental;
-
+  const maxSlippage = experimental.maxSlippage || 3;
```

Comment:
I was getting build errors post adding the `maxSlippage` field to lifecycle statusData. I had to update the input params and add the maxSlippage field to all setLifeCycleStatus calls

Reply:
This this should be
```
const { maxSlippage, setMaxSlippage } = useState(experimental.maxSlippage || 3);
```

That's because in the next PRs you are going to set the value of `maxSlippage`.